src/workshop-comments.njk

```diff
@@ -0,0 +1,58 @@
+---
+pagination:
+    data: collections.workshops
+    size: 1
+    alias: workshopItem
+permalink: "initiatives/{{ workshopItem.id }}/"
+---
+{% extends 'layouts/base.njk' %}
+
+{% set pageTitle = workshopItem.title %}
+
+{% block content %}
+	<article class="workshop-page">
+		<h1>{{ workshopItem.title | safe }}</h1>
+		{% if workshopItem.imageUrl %}
+			<img class="workshop-image" src="{{ workshopItem.imageUrl | safe }}" alt="{{ workshopItem.imageAlt }}">
+		{% endif %}
+
+		<div class="description">{{ workshopItem.fullDescription | safe }}</div>
+
+		<form id="comment-form">
+			<h3>Comments</h3>
+			<div class="form-field">
+				<span>*</span><p class="asterisk-def"> Indicates required field</p>
+			</div>
+
+			<div class="form-field">
+				<span>*</span><label for="name"><h4>Name</h4></label><span class="required required-name">Required</span>
+				<input id="name" type="text" name="name" placeholder="" aria-label="Enter your name">
+			</div>
+
+			<div class="form-field">
+				<span>*</span><label for="comment"><h4>Post a comment or question</h4></label><span class="required required-comment">Required</span>
+				<textarea id="comment" type="text" name="comment" placeholder="" aria-label="Enter your comment"></textarea>
+				<input type="hidden" name="workshopId" id="workshopId" value="{{ workshopItem.id }}">
+				<button id="post-comment" type="submit">Post a comment</button>
+			</div>
+		</form>
+
+		<div class="save-response post-success-message" role="status">
```

Comment:
The selector ".save-response" can be removed as each DOM already has its own selector identifying success or failure message.

Reply:
This selector is still needed for the styling of these elements in `_initiative.scss`

Reply:
This selector is still needed for the styling of these elements in `_initiatives.scss`

Reply:
The styling in the css can point to individual selectors: replace ".save-response" to ".submit-success-message, .submit-failure-message".